docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -125,6 +125,40 @@ imagePullSecrets:
 - name: "my-image-pull-secret"
 ```
 
+##### `nodeSelector`
+`nodeSelector` allows you to run the operator Pod on a node whose labels match the specified nodeSelector labels. You can use this optional feature if you want the operator Pod to run on a Node with particular labels. See [Assign Pods to Nodes](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) in the Kubernetes documentation for more details. This is not required if the operator Pod can run on any Node.
+
+Example:
+```
+nodeSelector:
+  disktype: ssd
+```
+
+##### `nodeAffinity`
+`nodeAffinity` is conceptually similar to `nodeSelector`, it allows you to constrain the operator Pod to be scheduled on a node with certain labels. Node affinity feature provides advanced capabilities to limit Pod placement on specific nodes. See  [Assign Pods to Nodes](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) in the Kubernetes documentation for more details. This is optional and not required if the operator Pod can run on any Node or when using `nodeSelector`.
```

Comment:
Node affinity feature provides advanced capabilities to limit... -> 'nodeAffinity` provides advanced capabilities to limit...
nodes ->Nodes